app/config/config.neon

```diff
@@ -51,6 +51,10 @@ parameters:
   monitor:  # connection to monitor
     address: wss://recodex.mff.cuni.cz:4443/ws  # URI of monitor
 
+  registration:
```

Comment:
I would probably name this option something like `local-registration` to distinguish it from general mechanism of registration to which even external ones can belong to.